src/components/BB-BUS/AZFilterElement.js

```diff
@@ -32,4 +32,10 @@ export const AZFilterElement = styled.View`
     css`
       margin-right: ${normalize(14)}px;
     `};
+
+  ${(props) =>
+    props.blackBackgroundForContrast &&
+    css`
+      background-color: ${colors.darkText};
```

Comment:
why have you decided to make it black here instead of `colors.overlayRgba` like in https://github.com/smart-village-solutions/smart-village-app-app/pull/777/files?

Reply:
I chose black to increase the contrast ratio. But in `colors.overlayRgba` we can get a contrast ratio of 6,9:1 and that should be enough. 👍

|darkText|overlayRgba|
|--|--|
<img width="535" alt="image" src="https://user-images.githubusercontent.com/11755668/233003107-c0a6282f-74c7-4482-86a2-7b451aadc5ce.png"> | <img width="539" alt="image" src="https://user-images.githubusercontent.com/11755668/233002995-1f58c9d3-d197-4833-b858-e0516ae020f8.png">

https://github.com/smart-village-solutions/smart-village-app-app/commit/c8b47067d02ff0a1dc6b6832058238818b26ea70